Assembly-CSharp/Global/btlseq.cs

```diff
@@ -795,21 +796,27 @@ public static Int32 SeqExecMessage(SEQ_WORK pSeqWork, BTL_DATA pMe)
 		{
 			btlseq.BattleLog("wOfs " + pSeqWork.CmdPtr.aa.Name);
 			UIManager.Battle.SetBattleTitle(pSeqWork.CmdPtr.aa.Name, 1);
+			// @todo i don't like this using the current animationName as the uniue key
+			int cmdNameIndex = FF9StateSystem.Battle.FF9Battle.btl_scene.header.TypCount + pSeqWork.CmdPtr.sub_no;
+
+			VoicePlayer.PlayBattleScriptVoice(cmdNameIndex, pSeqWork.CmdPtr.aa.Name);
 		}
 		else
 		{
+			string vaPath;
 			num = (UInt16)(num + (UInt16)FF9StateSystem.Battle.FF9Battle.enemy[pMe.bi.slot_no].et.mes);
-			btlseq.BattleLog("wMsg " + num);
+			string str;
```

Comment:
Variable name `str` is not a descriptive name. Seems like, this is a `battleText`.